analysis/tests/src/expected/Completion.res.txt

```diff
@@ -564,7 +564,7 @@ Completable: Cdecorator(react.)
 Package opens Pervasives.JsxModules.place holder
 Resolved opens 1 pervasives
 [{
-    "label": "component",
+    "label": "react.component",
```

Comment:
I don't understand this change, but it didn't affect autocomplete for `@react.component`

https://github.com/rescript-lang/rescript-vscode/assets/16160544/f0fa547e-056d-4219-8371-884242e15519